src/views/preview/preview.jsx

```diff
@@ -114,23 +140,35 @@ class Preview extends React.Component {
     }
     render () {
         return (
-            <PreviewPresentation
-                comments={this.props.comments}
-                creditInfo={this.props.credit}
-                editable={this.state.editable}
-                faved={this.props.faved}
-                favoriteCount={this.state.favoriteCount}
-                loveCount={this.state.loveCount}
-                loved={this.props.loved}
-                projectInfo={this.props.projectInfo}
-                remixes={this.props.remixes}
-                sessionStatus={this.props.sessionStatus}
-                studios={this.props.studios}
-                user={this.props.user}
-                onFavoriteClicked={this.handleFavoriteToggle}
-                onLoveClicked={this.handleLoveToggle}
-                onUpdate={this.handleUpdate}
-            />
+            this.state.inEditor ?
```

Comment:
I think `PreviewPresentation` should handle this logic since it's presentational, and we should just pass all the possible GUI props into that component. It can then handle how the GUI is actually rendered.  This will require moving the `<Page></Page>` piece into there, which is unlike our other views. I think that's ok — the same argument re: presentation applies.